Resources/Maps/bagel.yml

```diff
@@ -76381,7 +76381,7 @@ entities:
       type: Transform
     - tags: []
       type: Tag
-- proto: FoodCondimentBottleSmallHotsauce

```

Comment:
Prototype renames need an entry in migration.yml

Reply:
would i revert bagel change too then

Reply:
Yeah or emisse is going to murder you

Reply:
okey